End_of_year_Reallocations/credit_test.py

```diff
@@ -9,31 +9,35 @@
 #credit = pd.read_csv('Raw Data/RAW_credit.csv')
 credit = pd.read_csv('test_credit.csv')
 
-#print(financials.head(5))
-#print(credit.head(5))
+f_cols = ['LBU_CY_ON_AMT','LBU_CY_OA_AMT','LBU_CY_PR_AMT','LBU_PY_ON_AMT','LBU_PY_OA_AMT','LBU_PY_PR_AMT','LBU_SUBSCR','LBU_CAE_VAL']
```

Comment:
`f_cols` isn't a very descriptive variable name. Why not call this something more descriptive?